### ISSUE TITLE
Cold to Fire support broken with ngamahu's flame molten burst skill

### ISSUE THREAD

**Issue by Coraulten** [Openarl/PathOfBuilding#789]
The support was working fine in the previous patch of PoB, but since the update cold to fire is not adding its additonal cold damage or adding in its conversion for the molten burst skill that comes from the axe. I have both iterations of PoB up and i confirmed it works on the previous version. I also tried it on another skill (cyclone) and it seemed to work correctly.

**Comment by Openarl** [comment]
I cannot reproduce this; can you give me the build?

**Comment by Coraulten** [comment]
https://pastebin.com/wWYwbRp4 is the build

**Comment by Openarl** [comment]
It is definitely applying correctly.
![mbconv](https://user-images.githubusercontent.com/19189971/34472281-1973b418-efaa-11e7-900b-8e8e6b4ae73c.png)

**Comment by Coraulten** [comment]
I'm not getting that, the support is simply not applying its effect to molten burst for me.

**Comment by Openarl** [comment]
I'm sorry, but you are mistaken.

I didn't change your build in any way before posting those screenshots; I just loaded your build, went straight to the Calcs tab, and looked at those breakdowns. And they show the support applying 100% correctly; there's no doubt about that, And disabling the Cold to Fire gem results in the expected DPS loss (~35%), so again, no doubt there.

I also know that _your_ version of the program isn't calculating any differently from _my_ version, because the build code you provided contains the values of all the stats shown in the sidebar as calculated by your version, and they match what I'm seeing here.

You must be missing something here, but I'm not entirely sure what. How exactly are you determining that the support _isn't_ applying? Did you look at the breakdowns in the Calcs tab and compare them to what I posted? Are you checking the socket group tooltip (hover over the "Main Skill" selector in the sidebar) to see if it is listed?
